scripts/exgfs_wave_prdgen_bulls.sh

```diff
@@ -88,8 +88,8 @@
    echo ' '
    set_trace
    echo "${RUN} wave prdgen ${date} ${cycle} : bulletin untar error."
-   err=2;export err;err_chk
-   exit ${err}
+   export err=2
+   err_chk
```

Comment:
Is this a fatal error and should the echo be moved to the `err_chk`? @JessicaMeixner-NOAA